api/nativeMessaging/main.js

```diff
@@ -0,0 +1,80 @@
+// Copyright 2013 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+let port = null;
+
+const appendMessage = (text) => {
+  document.getElementById('response').innerHTML += '<p>' + text + '</p>';
+}
+
+const updateUiState = () => {
+  if (port) {
+    document.getElementById('connect-button').style.display = 'none';
```

Comment:
`.hidden = true` (or `false`) rather than setting style.

Reply:
I can substitute `hidden` property for setting `style` if that is critical.